cachito/web/api_v1.py

```diff
@@ -125,12 +125,21 @@ def create_request():
         tasks.fetch_app_source.s(request.repo, request.ref, request.id).on_error(error_callback),
     ]
     if 'gomod' in pkg_manager_names or auto_detect:
+        gomod_dep_overrides = [
+            dep_override.to_json()
+            for dep_override in request.dependency_overrides
+            if dep_override.type == 'gomod'
+        ]
         chain_tasks.append(
-            tasks.fetch_gomod_source.s(request.id, auto_detect=auto_detect).on_error(error_callback)
+            tasks.fetch_gomod_source.si(
```

Comment:
+1 to using immutable signatures. I think this makes the code easier to understand.